client/packages/system/src/Asset/ListView/AppBarButtons.tsx

```diff
@@ -58,6 +68,19 @@ export const AppBarButtonsComponent = ({
         >
           {t('button.export')}
         </LoadingButton>
+        {isCentralServer && (
+          <LoadingButton
+            isLoading={false}
+            startIcon={<InfoIcon />}
+            variant="outlined"
+            onClick={() => {
+              navigate(path);
+            }}
+            disabled={EnvUtils.platform === Platform.Android}
+          >
+            {t('button.manage-asset-log-reasons')}
+          </LoadingButton>
```

Comment:
No need to use a `LoadingButton` as there's no loading state:

```suggestion
          <BaseButton
            startIcon={<EditIcon />}
            variant="outlined"
            onClick={() => {
              navigate(path);
            }}
            disabled={EnvUtils.platform === Platform.Android}
          >
            {t('button.manage-asset-log-reasons')}
          </BaseButton>
```

I'm not loving the pencil but I don't think the info icon is appropriate for an edit area.. so wondered about this:

<img width="228" alt="image" src="https://github.com/msupply-foundation/open-msupply/assets/9192912/9b3ecc63-ebb5-4092-8650-be3b2b1dbf14">

Reply:
yes looks good